extension/content_script.js

```diff
@@ -322,28 +321,21 @@ Parameters:
 - fadeMs (int) ms for the fade animation speed.
 */
 function fadeTracker(fadeMs) {
-	fadeElement($("."+sentenceClass), fadeMs);
+	fadeElement(doc.getSentenceEls(tracker.getSentenceId()), fadeMs);
 	fadeElement($("."+keywordClass), fadeMs);
 }
 
 /*
 Stop all animations related to fading.
 */
 function stopFadeTracker() {
-	let sentenceStyle = trackerStyle.getSentenceStyle();
-	let keywordStyle = trackerStyle.getKeywordStyle(); 
+	// TODO: Fix transition animations.
```

Comment:
deleting some code here because
- i don't understand why the extra animation is needed
- this will have to be redone as part of "better sentence transition animation" anyway.